include/avif/internal.h

```diff
@@ -244,6 +254,9 @@ typedef struct avifCodec
     struct avifCodecInternal * internal;  // up to each codec to use how it wants
                                           //
     avifDiagnostics * diag;               // Shallow copy; owned by avifEncoder or avifDecoder
+                                          //
+    uint8_t operatingPointIndex;          // Operating point Index, defaults to 0.
```

Comment:
I suggest renaming this member `operatingPoint`. , because this corresponds to the `operatingPoint` variable in the AV1 spec.

Note: AV1 spec also has an `operating_point_idc[]` array and an `OperatingPointIdc` variable. The current name of this member, `operatingPointIndex`,  may potentially be misinterpreted as the `OperatingPointIdc` variable in the AV1 spec.